lib/publish.js

```diff
@@ -137,7 +137,12 @@ class Publish extends BaseCommand {
   publishConfigToOpts (publishConfig) {
     // create a new object that inherits from the config stack
     // then squash the css-case into camelCase opts, like we do
-    return flatten({...this.npm.config.list[0], ...publishConfig})
+    let mergedConfigs = {}
+    for (const config of this.npm.config.list)
+      mergedConfigs = {...mergedConfigs, ...config}
```

Comment:
`const mergedConfigs = Object.assign({}, ...this.npm.config.list, publishConfig)`

Reply:
After a little more thought here I wonder is this the right option?  If the defaults are last won't this make the defaults take priority?

What order is the config.list populated? We should answer that question and then see if applying it like this makes sense.

Reply:
oh true, i'm just matching the order in the PR in my suggestion.

if `publishConfig` is the defaults, then it should be `Object.assign({}, publishConfig, ...this.npm.config.list)`

Reply:
Yeah and I'm saying we should see what order `this.npm.config.list` itself, is populated.